### DISCUSSION TITLE
Questions: SAF requirements, access to github from z/OS, and disk storage requirements to install Z Open Tools?

### DISCUSSION THREAD

**Discussion by pjfarleyiii** [zopencommunity/meta#344] [General]
I have reviewed the docs provided here and nowhere did I find answers to these three questions from the perspective of an ordinary unprivileged developer on a legacy z/OS system:

Q1: Are there any special SAF permissions required for install?  
Q2: WRT github access from z/OS, is there a simple test one can do in a USS shell session that will tell you if the z/OS system has access to github repositories?  
Q3: How much disk space is required for (A) the downloads and (B) the final installed directory?  Many legacy shops severely limit the USS file space available to ordinary unprivileged developers.

Would someone please answer these questions?

**Comment by IgorTodorovskiIBM** [comment]
Hi there, thanks for the questions.

Q1: No special permissions are required, it's up to you where you want to install the tools. Most users are developers and set up the z/OS Open Tools under their $HOME directory (e.g. $HOME/zopen). However, you can set it up system wide by installing the tools into a location like `/usr/local/zopen`, which would obviously need write privileges to /usr/local.
Q2: You could probably try pinging github.com: `/bin/ping github.com`. If it responds back, then you should be good to go.
Q3: If you use the [getting started guide](https://zosopentools.github.io/meta/#/Guides/QuickStart?id=getting-zopen-setup) and start with `zopen-setup`, it's about 1GB. 

I think these would be good topics to add to the z/OS Open Tools FAQ: https://zosopentools.github.io/meta/#/Guides/FAQ.

**Reply by pjfarleyiii** [reply]
Thanks for the replies to my questions.  For the disk storage requirements, is that 1G for both downloads and final installed size?  This is important to people in a storage-restricted (by cost and/or fiat) environment.  Asking for more than you absolutely need has potentially negative consequences.

**Comment by lbdyck** [comment]
@pjfarleyiii No special SAF requirements.  For space this is what I have on my zfs for the zot tools
![image](https://github.com/ZOSOpenTools/meta/assets/42328411/5af96447-b88e-464a-a4b9-717942ff1577)

**Comment by MikeFultonDev** [comment]
@IgorTodorovskiIBM would be good to update this now - I expect the 'zopen' install minimum is now significantly less.
